js/sketchfab.js

```diff
@@ -119,7 +120,13 @@ SketchfabPlugin.prototype.onSubmit = function onSubmit() {
 	var modelId = url.substring(url.lastIndexOf('/') + 1);
 	var text = this.renderText(modelId);
 
-	this.insertAtCursor(text);
+	if ((this.mode || '').toLowerCase() === 'wysiwyg') {
```

Comment:
I think detecting whether the `composer` property exists on the editor object (like `if (editor.composer) { // do wysiwyg }`) is going to be more robust than manually checking for that string code.